### PR TITLE
Fixing issue getting quality control variable names

### PR DESCRIPTION
**Pull request by kenkehoe**
Initially I tried to not use the qc_ at start of quality control variables since that is only an ARM standard, and ARM does not follow its own standards. But there is no reasonable way to code up all the minor differences with the attributes of ARM QC variables, so I'm adding in some code to also look for qc_ in the variable name.

### CODE REVIEW

**Comment by kenkehoe** [comment]
Last time we had an error like this was because I moved some files around. I didn't do that this time. Do you have any further information on this error. I assume it's part of the HTML documentation generation?

**Comment by AdamTheisen** [comment]
@kenkehoe  did you do a fetch on the latest ACT updates?

**Comment by AdamTheisen** [comment]
@zssherman Pulling you in to see if you can see a fix for this that I did not

**Comment by kenkehoe** [comment]
I did pull in the latest version from ACT master before making the new branch. But Git and I have a strained relationship so who knows what actually happened.

**Comment by AdamTheisen** [comment]
Pulling in @rcjackson  as well to see if he can figure out what's going on here.

File: act/qc/clean.py [hunk 1]
```diff
@@ -10,10 +19,19 @@ def __init__(self, xarray_obj):
...
-    def matched_qc_variables(self):
+    def matched_qc_variables(self, check_arm_syntax=True):
         """
         Find variables that are QC variables and return list of names.
 
+        Parameters
+        ----------
+        check_arm_syntax : boolean
+            ARM ueses a standard of starting all quality control variables
+            with "qc_". This is a more robust method of getting the quality
```
Comment (rcjackson) [inline act/qc/clean.py]: The underscore is your issue here. In RST, an underscore is a hyperlink. This needs to be "qc\_".

**Comment by AdamTheisen** [comment]
Thanks @rcjackson !  @kenkehoe I pushed the fix.  Will merge.

**Comment by kenkehoe** [comment]
Hmmm OK. Good catch. I'll try to remember this for the future.

**Pull request merged**